tools/mo/requirements_tf.txt

```diff
@@ -1,5 +1,5 @@
-tensorflow~=2.5.0; python_version <= "3.6"
-tensorflow~=2.5.3; python_version > "3.6"
+tensorflow>=1.15.5,<2.6; python_version <= "3.6"
+tensorflow>=1.15.5,<2.6; python_version > "3.6"
```

Comment:
Looks like python_version check can be removed and single TF version line left.

Reply:
Thanks! done